### PR TITLE
First stage desktop version

### PR DESCRIPTION
**Pull request by joramot**
- Create desktop version of my portfolio according to the requirements list.
- First implementation of media queries
- Implement Animations and transitions

### CODE REVIEW

File: css/style.css [hunk 1]
```diff
@@ -0,0 +1,845 @@
...
+    background-color: black;
+    background-image: none;
+  }
+
+  .head-information h1 {
+    font-size: 4rem;
+  }
+
+  .header-about p {
+    margin-left: 142px;
```
Comment (Nyame-Wolf) [inline css/style.css]: I suggest you fix the styles of your page. It uses the template, but your CSS is broken. This is how your page appears both on mobile and Desktop
Screenshot 2023-01-21 at 07 56 49
Screenshot 2023-01-21 at 07 57 05

**Review by Nyame-Wolf — CHANGES_REQUESTED** [review]
# STATUS: REQUIRED CHANGES :recycle: 

Hi @joramot  :wave: 

Your project page is looking fantastic so far :star_struck: ! 

However, there is still some room for improvement, that can help you have better code and make your project professional :sunglasses: 



## Highlights :sparkles:

- Great work using the right GitHub Flow :100: 
- Great work using the  Figma design ☑️ 
- Commits are descriptive and meaningful :+1: 
- Project Documentation (PR, README) is professional :necktie: 

Congrats on your work so far! :clap:  

## Required Changes ♻️

There are some issues that you still need to work on to go to the next project, but you are almost there! 
_For more information, check the comments under the review :arrow_down:_

## Optional suggestions :ballot_box_with_check: 

_Every comment with the **[OPTIONAL]** prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you take them into account as they can improve the quality of your code :100:_

Happy coding and keep it up! :raised_hands: 

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
  Please, remember to tag me **@Nyame-Wolf**  in your question so I can receive the notification.

  **_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------

  _As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)

**Review by joramot — COMMENTED** [review]
Restructure all sections and rename CSS classes whit short names.
Implement Media Queries in all sections.

**Review by BregornOriginal — APPROVED** [review]
## Hi @joramot 

## Your project is complete!  There is nothing else to say other than... it's time to merge it 🥳 🥳 🥳 

## Congratulations! 💯 💯 💯 

giphy

## Highlights

✅ Linters are passing
✅ Good commit history
✅ Good Pull Request description
✅ Good work making the changes requested by the previous reviewer

## Optional suggestions
Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application.
Cheers and Happy coding!:aplauso::aplauso::aplauso:
Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

Please, remember to tag me in your question so I can receive the notification.
------
As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)[.](https://airtable.com/shrQAqnBwek5a0O0s)._)

**Review by dhy-612 — COMMENTED** [review]
You have done great work.

I didn't find any issue in the animation part. 👍 👍 

Just follow up for next part, 
send it for review 😀

**Pull request merged**